include/VSSRequestValidator.hpp

```diff
@@ -29,25 +29,25 @@
 #include "ILogger.hpp"
 
 
-
-
 class VSSRequestValidator {
     public:
         VSSRequestValidator(std::shared_ptr<ILogger> loggerUtil);        
         ~VSSRequestValidator();
 
         void validateGet(jsoncons::json &request);
         void validateSet(jsoncons::json &request);
-        void validateUpdateTree(jsoncons::json &request);
+        void validateUpdateMetadata(jsoncons::json &request);
+        void validateUpdateVSSTree(jsoncons::json &request);
+
         std::string tryExtractRequestId(jsoncons::json &request);
 
     private:
-        std::shared_ptr<jsoncons::jsonschema::json_schema<jsoncons::json> > getSchema;
-        std::shared_ptr<jsoncons::jsonschema::json_schema<jsoncons::json> > setSchema;
-        std::shared_ptr<jsoncons::jsonschema::json_schema<jsoncons::json> > updateTreeSchema;
-        jsoncons::jsonschema::json_validator<jsoncons::json> *getValidator;
-        jsoncons::jsonschema::json_validator<jsoncons::json> *setValidator;
-        jsoncons::jsonschema::json_validator<jsoncons::json> *updateTreeValidator;
+        class MessageValidator;
+        VSSRequestValidator::MessageValidator *getValidator;
```

Comment:
there pointers are perfekt as unique
```suggestion
        unique_ptr<MessageValidator> getValidator;
```